cmd/kubemci/kubemci.go

```diff
@@ -29,4 +29,5 @@ func main() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+	fmt.Println("Exiting.")
```

Comment:
left over?

Reply:
No, it was intentional because there is so much output, and the current last line is not very satisfying. I want it to say "success!" or "error!". I didn't dig enough to find where that branch would be, so this seemed better than nothing.

Reply:
fine as is in this PR.
I think we can say "Success" here and "Error" inside the if on line 28

Reply:
that does look like the obvious solution. I thought I tried that and it didn't work as one would expect. But, I figured out that the success/error check needs to be in create.go:NewCmdCreate().